projects/components/src/select/select.component.ts

```diff
@@ -224,6 +224,9 @@ export class SelectComponent<V> implements ControlValueAccessor, AfterContentIni
   @Input()
   public showBorder: boolean = false;
 
+  @Input()
+  public showClearSelected: boolean = true;
```

Comment:
Please default to false for compatibility

Reply:
Changing it to `false`.

cc: @anandtiwary

Reply:
Yes this makes sense.